page-builder/hooks/custom-row-settings.md

```diff
@@ -86,3 +86,23 @@ To add a custom style (like text-align: center;), append the style to ``$attribu
 To add a new attribute, add both a new key and value to ``$attributes``. Example: ``$attributes['data-video-id'] = $args['video-id'];`` will add ``data-video-id="9"`` to the row element.
 
 Note that the filter for ``siteorigin_panels_row_style_attributes`` needs to be called after Page Builder loads. In my project I added the filter inside a function which is triggered on ``plugins_loaded``.
+
+### Processing Styles
+Prior to rendering the contexts styles, it's possible to access and alter styles of the current context using filters. This allows for you to migrate and alter data as needed. Changes aren't however stored until the user saves the page/widget. There is two filters that allow for this. `siteorigin_panels_general_current_styles`, which is a general catch all filter, and `siteorigin_panels_general_current_styles_{type}`, `{type}` being the currently viewed type.
```

Comment:
How about:

Before rendering the context's styles, it's possible to access and alter the styles of the current context using filters. This allows you to migrate and alter data as needed. Changes aren't, however, stored until the user saves the page/widget. There are two filters that allow for this. `siteorigin_panels_general_current_styles`, which is a general catch-all filter, and `siteorigin_panels_general_current_styles_{type}`, `{type}` being the currently viewed type.

Reply:
Looks good.